src/bp/ui-admin/src/Pages/Server/Alerting.tsx

```diff
@@ -1,16 +1,16 @@
+import moment from 'moment'
+import ms from 'ms'
 import React, { Component } from 'react'
-import { connect } from 'react-redux'
 import { IoIosArchive } from 'react-icons/io'
-import { Label, Row, Col, Jumbotron } from 'reactstrap'
+import { connect } from 'react-redux'
 import Select from 'react-select'
-import moment from 'moment'
-import ms from 'ms'
-
-import SectionLayout from '../Layouts/Section'
-import IncidentsTable from '../Components/Monitoring/IncidentsTable'
-import LoadingSection from '../Components/LoadingSection'
-import { fetchIncidents } from '../../reducers/monitoring'
-import CheckRequirements from '../Components/CheckRequirements'
+import { Col, Jumbotron, Label, Row } from 'reactstrap'
+import { fetchIncidents } from '~/reducers/monitoring'
+import PageContainer from '~/App/PageContainer'
+import SplitPage from '~/App/SplitPage'
```

Comment:
What's that ?

Reply:
Replaces the SectionLayout component which is used in the old admin page. It gives the menu on the right side with actions